src/coreComponents/python/modules/geosx_xml_tools_package/geosx_xml_tools/__main__.py

```diff
@@ -5,32 +5,34 @@
 
 
 def main():
-  """Entry point for the geosx_xml_tools console script
+    """Entry point for the geosx_xml_tools console script
```

Comment:
Most of the changes below are due to changing the indentation level to match PEP8 standards.  Hiding whitespace changes in the diff is highly recommended.